### ISSUE TITLE
Port, PreferredBackupWindow and PreferredMaintenanceWindow are not supported for serverless DBs

### ISSUE THREAD

**Issue by ptmclean** [cfn-modules/rds-aurora-serverless#3]
Hi,

I think there is a problem with the updating existing stacks with this template. I haven't been able to find the docs that say why this is the case but updates with the following 3 lines in the AWS::RDS::DBCluster resource fail.
```
      Port: !FindInMap [EngineMap, !Ref Engine, Port]
      PreferredBackupWindow: !Ref PreferredBackupWindow
      PreferredMaintenanceWindow: !Ref PreferredMaintenanceWindow
```

Basically to reproduce you just need to deploy a stack twice. The second update will fail with..
```
You currently can't modify BackupWindow with Aurora Serverless.
```
I'm not sure if this is a bug with the Amazon or not. It's weird that these parameters are permitted on create but not update.

**Comment by michaelwittig** [comment]
that might be a simple question, just making sure that we don't miss it: You are not changing the PreferredBackupWindow parameter when updating, right?

**Comment by ptmclean** [comment]
I played a little more with this, this morning. I thought it was reproducible without any change but it looks like you do need to change something in the template, but you don't need to change the Port PreferredBackupWindow or the PreferredMaintenanceWindow.

I created a repro case [here](https://github.com/ptmclean/aurora-repro). I'll see how things behave when the properties are not set and if the stack behaves itself without them then I'll PR to remove.

**Comment by ptmclean** [comment]
I took another look at this and I'm not sure of the best way to proceed. PreferredBackupWindow could likely be removed assuming this template is only looking to support severless aurora as I believe it is ignored anyway.

The other 2 aren't ignored so removing Port and PreferredMaintenanceWindow remove the ability to change the defaults which is not ideal. Any ideas?

**Comment by michaelwittig** [comment]
Do you have access to AWS support? It sounds like a bug that CloudFormation tries to change the BackupWindow if the parameter does not change?

**Comment by ptmclean** [comment]
You are right, this is an issue with either the cli CloudFormation. I don't have access to the premium support, but I raised this [question](https://forums.aws.amazon.com/thread.jspa?threadID=300406) in the forum.

It would be great if someone has access to raise a support ticket, but this is not an issue with this repo so I'll close this issue.

**Comment by michaelwittig** [comment]
thanks for the update! Can you post the cli command you use to update the stack?

**Comment by michaelwittig** [comment]
I'm seeing the same issue now. https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Modifying.html mentions that "Although the preferred backup window can be set for Aurora, the setting is ignored. Aurora backups are continuous and incremental."

Unfortunately, I can not remove the `PreferredBackupWindow` parameter because this results in "You currently can't modify MaintenanceWindow with Aurora Serverless."

**Comment by michaelwittig** [comment]
Support confirmed that this is a bug. Currently a possible workaround is to comment out the following properties:

    -  "PreferredBackupWindow", 
    - "PreferredMaintenanceWindow" and 
    - "Port"

**Comment by michaelwittig** [comment]
fixed in #8
